Modules/IO/NRRD/src/itkNrrdImageIO.cxx

```diff
@@ -1065,9 +1082,23 @@ NrrdImageIO::Write(const void * buffer)
     else
     {
       // not a NRRD field packed into meta data; just a regular key/value
-      std::string value;
-      ExposeMetaData<std::string>(thisDic, *keyIt, value);
-      nrrdKeyValueAdd(nrrd, keyIt->c_str(), value.c_str());
+      // convert to string and dump to the file
+      // const char *tname = thisDic.Get(*keyIt)->GetNameOfClass();
+      std::ostringstream dump;
+      if (_dump_metadata_to_stream<std::string>(thisDic, *keyIt, dump) ||
```

Comment:
Shouldn't `string` be last in this list?

Reply:
i thought that most of metadata is strings, so if it's the first - it will get processed faster.